### PR TITLE
Checklist status field missing options

### LINKED ISSUE #484: GtChecklistStatus missing all choices in v1.2.7 for projects
After changes to _jsontemplate in PR #473 all new projects are missing options for the field GtChecklistStatus 

This is the line causing the issue
https://github.com/Puzzlepart/prosjektportalen365/blob/35f2666a243c3cd6c32550dfa14545d0528f138b/Templates/_JsonTemplate.json#L693

Problem is we are redefining GtChecklistStatus for this list, where we previously were only overriding the Title column

Mitigating actions: 
1. revert change for L693 and push json file to any an all new installations we have done
2. create a delta pnp template to fix any projects created where GtChecklistStatus are missing choises
3. release new installation
4. inform customers on upgrade path

### PR DESCRIPTION
**Pull request by okms**
Removing definition of field GtChecklistStatus on list Fasesjekkliste, as this definition overrides the sitefield and causes this field to be missing its options

### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [x] Make sure you are making a pull request against the **dev** branch (left side). Also you should start *your branch* off *dev*.
- [x] Check the commit's or even all commits' message 
- [x] Check if your code additions will fail linting checks
- [ ] Remember: Add PR description to [CHANGELOG](https://github.com/Puzzlepart/prosjektportalen365/blob/dev/CHANGELOG.md) with the ID that matches this PR

### Description

- 
### How to test
- Add _JsonTemplate.json to a tenant currently using pp365 1.2.7
- create a new project
- observe that the phasechecklist has the "status" field and that this field has the correct options

### Relevant issues (if applicable)

Closes #484

### CODE REVIEW

File: Templates/_JsonTemplate.json [hunk 1]
```diff
@@ -690,7 +690,7 @@
                 "EnableVersioning": true
             },
             "Fields": [
```
Comment (Remi749) [inline Templates/_JsonTemplate.json:692]: Remove 692-694. Title overriding never worked as mentioned in issue

Comment (olemp) [reply Templates/_JsonTemplate.json:692]: I suggest removing those lines aswell @okms

**Review by olemp — APPROVED** [review]

**Comment by olemp** [comment]
And don't remove package-lock please @okms - it has nothing to do with the change and is needed for `npm ci`

https://docs.npmjs.com/cli/ci.html/

**Review by olemp — CHANGES_REQUESTED** [review]
Don't remove package-lock please @okms - it has nothing to do with the change and is needed for npm ci

**Comment by okms** [comment]
looks like a git goof, did not intentionally delete it. i did have a npm run that failed, so that might have cleared out package lock

**Comment by olemp** [comment]
> looks like a git goof, did not intentionally delete it. i did have a npm run that failed, so that might have cleared out package lock

Ahh, it seemed deleted but it's just a change

**Pull request merged**